src/demo/containers/App.tsx

```diff
@@ -109,96 +109,65 @@ export default class App extends React.Component<{}, AppState> {
           <Route
             exact={true}
             path="/drugUploadTool"
-            render={props => (
+            render={props => {
+              let searchParamsProps: any = {}
+              // https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams -- needs polyfill for ie11
+              const searchParams: any = new URLSearchParams(props.location.search)
+              const iter = searchParams.entries()
+              let result = iter.next()
+              while (!result.done) {
```

Comment:
why not forEach here?

Reply:
Didn't see that method as an option, used it now, thanks!